src/elona/ui.cpp

```diff
@@ -612,6 +612,27 @@ void render_clock()
 
 
 
+void render_digital_clock()
+{
+    // 24 hour digital clock, 57 pixels wide
+    font(16 - en * 2);
+    bmes(""s + game_data.date.hour + u8":"s + game_data.date.minute + u8":"s + game_data.date.second,
```

Comment:
I think second is not useful, is it?

Reply:
Are you talking about font size? I have added that because I wanted clock to be a little bigger than date and weather.

Reply:
It can be useful for people who want to measure how much time exactly passes with their current speed/equipment load per turn. Usually a single turn is less than a minute.

Reply:
It makes sense.